packages/govern-subgraph/src/GovernQueue.ts

```diff
@@ -62,8 +62,10 @@ export function handleScheduled(event: ScheduledEvent): void {
   // should be impossible to get at this stage
   container.config = queue.config
   container.queue = queue.id
+  let config = ConfigEntity.load(queue.config)
+  let scheduleDeposit = CollateralEntity.load(config.scheduleDeposit)
```

Comment:
I was thinking of creating `loadOrCreate` helper here... but these entities relies on values that can't be left `null`. If we want to still create helpers, we should just raise an error.